docs/common/common_examples.rst

```diff
@@ -755,3 +755,194 @@ EN 6. Authorization Request
       },
       "state": "fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd"
   }
+
+.. _Esempio_EN7:
+
+EN 7. Metadata Policy
++++++++++++++++++++++
+
+The following example shows a metadata policy in the Entity Statement provided by a TA and related to an RP
+
+.. code-block:: python
+
+    "metadata_policy": {
+        "openid_relying_party": {
+            "jwks": {
+                "one_of": [{
+                    "kty": "RSA",
+                    "use": "sig",
+                    "n": "…",
+                    "e": "AQAB",
+                    "kid": "5NNNoFS3YnC9tjiCaivhWLVUJ3AxwGGz_98uRFaqMEEs"
+                }]
+            }
+            "grant_types": {
+                "one_of": ["authorization_code", "refresh_token"]
+            }
+            "id_token_signed_response_alg": {
+                "one_of": ["RS256", "RS512", "ES256", "ES512", "PS256", "PS512"]
+            }
+            "id_token_encrypted_response_alg": {
+                "one_of": ["RSA-OAEP", "RSA-OAEP-256", "ECDH-ES", "ECDH-ES+A128KW", "ECDH-ES+A256KW"]
+            }
+            "id_token_encrypted_response_enc": {
+                "one_of": ["A128CBC-HS256", "A256CBC-HS512"]
+            }
+            "userinfo_signed_response_alg": {
+                "one_of": ["RS256", "RS512", "ES256", "ES512", "PS256", "PS512"]
+            }
+            "userinfo_encrypted_response_alg": {
+                "one_of": ["RSA-OAEP", "RSA-OAEP-256", "ECDH-ES", "ECDH-ES+A128KW", "ECDH-ES+A256KW"]
+            }
+            "userinfo_encrypted_response_enc": {
+                "one_of": ["A128CBC-HS256", "A256CBC-HS512"]
```

Comment:
all the previous on_of should be changed to subset_of

Reply:
done